spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/tf/Const.scala

```diff
@@ -29,8 +29,8 @@ private[bigdl] trait WithoutInput
  * @param value the constant tensor to be returned in forward
  */
 @SerialVersionUID(-4008935551091949324L)
-private[bigdl] class Const[T: ClassTag](value: Tensor[T])(implicit ev: TensorNumeric[T])
-  extends AbstractModule[Activity, Tensor[T], T] with WithoutInput {
+private[bigdl] class Const[T: ClassTag](value: Tensor[_])(implicit ev: TensorNumeric[T])
```

Comment:
Is it conflicted with https://github.com/intel-analytics/BigDL/pull/1569/files#diff-e9e9bf4d715a7a08d0391cc28feede35 ?